contrib/99designs/gqlgen/example/server/server.go

```diff
@@ -0,0 +1,25 @@
+package main
```

Comment:
Can we get rid of this entire folder? I think you can just import something from https://github.com/99designs/gqlgen/tree/master/example for the example. It's not acceptable to have this in our public API. Any user will be able to import this. If it _must_ be here, put it in an `internal` folder; but I'm pretty sure we can avoid it.

Reply:
👍